sdks/python/apache_beam/io/gcp/experimental/spannerio.py

```diff
@@ -109,20 +111,74 @@
 
 ReadFromSpanner takes this transform in the constructor and pass this to the
 read pipeline as the singleton side input.
+
+Writing Data to Cloud Spanner.
+
+The WriteToSpanner transform writes to Cloud Spanner by executing a
+collection a input rows (WriteMutation). The mutations are grouped into
+batches for efficiency.
+
+WriteToSpanner transform relies on the WriteMutation objects which is exposed
+by the SpannerIO API. WriteMutation have five static methods (insert, update,
+insert_or_update, replace, delete). These methods returns the instance of the
+_Mutator object which contains the mutation type and the Spanner Mutation
+object. For more details, review the docs of the class SpannerIO.WriteMutation.
+For example:::
+
+  mutations = [
+                WriteMutation.insert(table='user', columns=('name', 'email'),
+                values=[('sara'. 'sara@dev.com')])
+              ]
+  _ = (p
+       | beam.Create(mutations)
+       | WriteToSpanner(
+          project_id=SPANNER_PROJECT_ID,
+          instance_id=SPANNER_INSTANCE_ID,
+          database_id=SPANNER_DATABASE_NAME)
+        )
+
+You can also create WriteMutation via calling its constructor. For example:::
+
+  mutations = [
+      WriteMutation(insert='users', columns=('name', 'email'),
+                    values=[('sara", 'sara@example.com')])
+  ]
+
+For more information, review the docs available on WriteMutation class.
+
+WriteToSpanner transform also takes 'max_batch_size_bytes' param which is set
+to 1MB (1048576 bytes) by default. This parameter used to reduce the number of
+transactions sent to spanner by grouping the mutation into batches. Setting
+this either to smaller value or zero to disable batching.
+
```

Comment:
Please add a note to the following

> Unlike the Java connector, this connector  _does not_ create batches of transactions sorted by table and primary key. 

This can be a feature which is added later, I would not let it block this PR. 
See https://medium.com/google-cloud/cloud-spanner-maximizing-data-load-throughput-23a0fc064b6d for more info.